lib/liquider/ast.rb

```diff
@@ -67,5 +77,9 @@ def visit(visitor)
   NullNode = Node.new_type(:null) do
     def visit(visitor)
     end
+
+    def op?
```

Comment:
In theory we could make it more intelligent and avoid branches that will never `op`